jquery.flot.js

```diff
@@ -1022,7 +1021,7 @@ Licensed under the MIT license.
                     label = t.label.name;
                 }
 
-                var info = surface.getTextInfo(layer, label, font, null, maxWidth);
+                var info = surface.getTextInfo(layer, label, font);
```

Comment:
The fix is simply to not pass a max width when measuring the text. That parameter is useful when wrapping is needed.